christfellowshipchurch/apollos-api
# Added support for DefinedValues for `themeResolver`

### About
This PR allows to use the Digital Platform Themes Define Type in Rock for our `themeResolver` for content items.

### Test Instructions
* run api locally
* update theme to a content item
* see it locally on web
